File: two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        int[] rsltArr = null;
        for(int i = 0; i < nums.length -1 ; i++) {  
            int num = nums[i];
            for(int j = (i + 1); j < nums.length; j++) {
               
                if((num + nums[j]) == target) {
                    return new int[]  {i,j};
                    
                } 
            }
        }
        
        return rsltArr;
    }
    
}

File: two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        nums_dict = {}
        nums_size = len(nums)

        for i in range(0, nums_size) :
            dict_key = target - nums[i]

            if dict_key in nums_dict.keys() :
                return [nums_dict[dict_key], i]
            else :
                nums_dict[nums[i]] = i


        return []

